apps/storybook/stories/FriendBlock.stories.tsx

```diff
@@ -23,6 +21,8 @@ export const Female: Story = {
   args: {
     name: `Elizabeth Webb`,
     gender: `female`,
-    blurb: Male.args?.blurb,
+    blurb: Male.args.blurb,
```

Comment:
...it allows us to not have to add optional chaining and non-null assertions everywhere, and to get a type error if we forget a prop.  for some components with function props, this might be a little extra boilerplate, but i think it will be worth it.  no more `undefined`s in stories, those are now a typecheck/build-time error.

Reply:
Wow that's pretty awesome, I always thought that would be nice